modules/users/client/views/authentication/confirm-email.client.view.html

```diff
@@ -16,6 +16,7 @@
 
           <div class="text-center form-group" ng-if="!confirmEmail.success">
 
+            <a ng-if="confirmEmail.email" href="mailto:{{ confirmEmail.email }}"><h4>{{ confirmEmail.email }}</h4></a>
```

Comment:
This could be simply `<h4 ng-if="::confirmEmail.email" ng-bind="::confirmEmail.email"></h4>`

(`::` is a [one-time binding](http://blog.thoughtram.io/angularjs/2014/10/14/exploring-angular-1.3-one-time-bindings.html))